test/e2e/upgradetest/framework/framework.go

```diff
@@ -138,6 +139,45 @@ func (f *Framework) UpgradeOperator() error {
 	return err
 }
 
+func (f *Framework) WaitOperatorElected() error {
```

Comment:
we need to comment on this. here, we assume the operator is deployed by k8s deployment? (or where is the name selector comes from?)